internal/session/service_close_conections_test.go

```diff
@@ -0,0 +1,97 @@
+package session
```

Comment:
Looks like there is a typo in this filename, I think it should be `service_close_connections_test.go`.